packages/tasks/src/library-ui-elements.ts

```diff
@@ -541,6 +541,14 @@ transcriptions = asr_model.transcribe(["file.wav"])`,
 
 const mlAgents = (model: ModelData) => [`mlagents-load-from-hf --repo-id="${model.id}" --local-dir="./downloads"`];
 
+const sentis = (model: ModelData) => [
+	`string modelName = "[Your model name here].sentis";
```

Comment:
Out of curiosity, what is the "model name" referred to here? Can it be inferred from the repo_id on the Hub? Asking because if you can infer it from `ModelData`, that would make the snippet even better. In `ModelData` class you have access to the model id, the tags, the config, etc.

(if not possible, just ignore my comment :) )

Reply:
Hi, unfortunately this can't be inferred from the ModelData as in a lot of situations (e.g. Whisper or Stable Diffusion) it requires multiple *.sentis files. So I think the best we can do is give a general example here. We also provide a full C# code along side each model which most people will use instead of the snippet. 

I think there are two other option which might be better?: (1) Just remove this snippet entirely. (2) Do some fancy code which outputs the entire c# file as the snippet - which is probably overkill.

Reply:
Thanks for the reply @pauldog! In the meantime I saw the discussion you've already had on slack about it so I'll defer to them for what's the best solution.